app/builders/text_field.rb

```diff
@@ -25,7 +26,11 @@ def labelled_text_field
   end
 
   def value
-    f.text_field attribute, input_options
+    if @multiline
+      f.text_area attribute, input_options.merge({rows: @multiline})
+    else
+      f.text_field attribute, input_options
+    end
```

Comment:
I didn't want to do it, because of the already large complexity of the custom form builder. But yeah, I can spend some more time on this.